File: src/libs/ThirdParty.VBSuppliers/Properties/JonView/IJonViewSettings.vb
Imports ThirdParty.Abstractions

Public Interface IJonViewSettings
    ReadOnly Property AllowCancellations(tpAttributeSearch As IThirdPartyAttributeSearch) As Boolean
    ReadOnly Property OffsetCancellationDays(ByVal tpAttributeSearch As IThirdPartyAttributeSearch, IsMandatory As Boolean) As Integer
    ReadOnly Property URL(tpAttributeSearch As IThirdPartyAttributeSearch) As String
    ReadOnly Property Password(tpAttributeSearch As IThirdPartyAttributeSearch) As String
    ReadOnly Property UserID(tpAttributeSearch As IThirdPartyAttributeSearch) As String
    ReadOnly Property ClientLoc(tpAttributeSearch As IThirdPartyAttributeSearch) As String
End Interface

File: src/libs/iVectorOne.Suppliers/JonView/IJonViewSettings.cs
namespace ThirdParty.CSSuppliers.JonView
{
    public interface IJonViewSettings
    {
        bool AllowCancellations(IThirdPartyAttributeSearch tpAttributeSearch);
        int OffsetCancellationDays(IThirdPartyAttributeSearch tpAttributeSearch, bool IsMandatory);
        string GenericURL(IThirdPartyAttributeSearch tpAttributeSearch);
        string Password(IThirdPartyAttributeSearch tpAttributeSearch);
        string User(IThirdPartyAttributeSearch tpAttributeSearch);
        string ClientLoc(IThirdPartyAttributeSearch tpAttributeSearch);
    }
}